.github/workflows/atlantis-image.yml

```diff
@@ -202,7 +202,7 @@ jobs:
     runs-on: ubuntu-24.04
     strategy:
       matrix:
-        image_type: [alpine, debian]
+        image_type: [alpine, alpine-slim-terraform, alpine-slim-tofu, debian]
```

Comment:
this test is failing now b/c goss file is setup to expect images to always have both tofu and terraform

will need to adjust expectations to account for alpine-slim* images no longer having both bins installed

looking at https://goss.readthedocs.io/en/stable/cli/ a couple paths for this could be:
- different `--gossfile` parm for the slim images
- different `--vars` param for the slim images
- something like this to only run the 2 test cases in question when relevant (set via env var):
```
{{if eq .Env.OS "centos"}}
  # This test is only when $OS environment variable is set to "centos"
  libselinux:
    installed: true
{{end}}
```